### PR TITLE
Completed Exercise13

### PR DESCRIPTION
**Pull request by tusharm-cuelogic**
# Understands following points in exercise 13
# How to import/add module to python script.
# How to assigned argument to variables.
# How to pass arguments to python script.

### CODE REVIEW

File: ex13.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from sys import argv
```
Comment (onkartibe) [inline ex13.py:1]: see the list of different libraries available in python. Also list popular libraries here.

Comment (tusharm-cuelogic) [inline ex13.py:1]: request - http library
wxPython - gui toolkit for python
SQLAlchemy - database library
BeautifulSoup - xml and html parsing library
 NumPy - math functionalities to python
sys - provides access to some variables used or maintained by the interpreter and to functions

Comment (onkartibe) [inline ex13.py:1]: http://pythontips.com/2013/07/30/20-python-libraries-you-cant-live-without/

http://pypi-ranking.info/alltime

Comment (onkartibe) [inline ex13.py:1]: Keep this in list. In free time go through the list.

**Pull request merged**